src/class_resolver/api.py

```diff
@@ -118,6 +119,32 @@ def get_option(self, *flags: str, default: Optional[str] = None, **kwargs):
             **kwargs,
         )
 
+    def ray_tune_search_space(self, kwargs_search_space: Optional[Mapping[str, Any]] = None):
+        """Return a search space for ray.tune."""
+        try:
+            import ray.tune
+        except ImportError:
+            raise ImportError(dedent(
+                """
+                To use ray_tune_search_space please install ray tune first.
+
+                You can do so by selecting the appropriate install option for the package
+
+                    pip install class-resolver[ray]
+
+                or by manually installing ray tune
+
+                    pip install ray[tune]
+                """,
+            )) from None
+        if kwargs_search_space is None:
+            return ray.tune.choice(sorted(self.lookup_dict.keys()))
+
+        return dict(
+            query=ray.tune.choice(sorted(self.lookup_dict.keys())),
```

Comment:
you use this same line twice but just return different ways, why not save it as a variable before the first `if kwargs_search_space is None:` check

Reply:
c67c5bf